src/main/java/org/example/homework6/Calculator.java

```diff
@@ -0,0 +1,77 @@
+package org.example.homework6;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class Calculator {
+    String decisionValue;
+    Scanner scanner = new Scanner(System.in);
+
+    public void calculator() {
+        double firstNum;
+        double secondNum;
+
+        do {
+            System.out.print("Enter a number:");
+            try {
+                firstNum = scanner.nextDouble();
+                scanner.nextLine();
+            } catch (InputMismatchException e) {
+                System.out.println("Invalid value! Number required.");
+                scanner.nextLine();
+                continue;
+            }
+
+            System.out.print("Enter an operation (+, -, *, /):");
+            String arithmeticOpr = scanner.nextLine();
+
+            System.out.print("Enter another number:");
+            try {
```

Comment:
You have 2 similar pieces of code for 2 doubles. you can extract them to separate methods.